Makefile

```diff
@@ -58,6 +58,9 @@ test-ci:
 	make bootstrap
 	make test-only
 
+update-babel-types-docs:
```

Comment:
I actually called it `check-babel-types-docs` before, but renamed it as the error sounded better when it complained about `update...` .. but I'll change it back.